exp/runtime/hooks/api/v1alpha1/topologymutation_variable_types.go

```diff
@@ -35,6 +35,9 @@ type ClusterBuiltins struct {
 	// Namespace is the namespace of the cluster.
 	Namespace string `json:"namespace,omitempty"`
 
+	// UID is the unqiue identifier of the cluster.
```

Comment:
Let's document this here: https://github.com/kubernetes-sigs/cluster-api/blob/v1.7.3/docs/book/src/tasks/experimental-features/cluster-class/write-clusterclass.md#builtin-variables

Reply:
Let's also add it here: https://github.com/kubernetes-sigs/cluster-api/blob/8965f4c50dd26d2336e4b4483b44738e17bfb59f/internal/webhooks/patch_validation.go#L472